src/insert.rs

```diff
@@ -143,6 +109,12 @@ impl<T> Insert<T> {
         self
     }
 
+    /// Similar to [Client::with_option], but for this particular INSERT statement only.
```

Comment:
Now all references to the code are wrapped into a backtick (there is a difference, call `cargo doc` and make sure of it).

Let's keep it consistent:
```
[`Client::with_option`]
```
or
```
[`Client::with_option()`]
```

Reply:
Can we make rustfmt or something else do that?

Reply:
I've found only https://rust-lang.github.io/rust-clippy/master/index.html#/doc_link_with_quotes that's useful, but doesn't cover this case